### PR TITLE
Geting apptags

### PR DESCRIPTION
**Pull request by mayabrandi**
Front end adjustments based on the new application tag collection.

In some plots, samples are supposed to be grouped by application tag category. The application tag category is hold by the new application tag collection.

This PR contains the front end adjustments to make this happen. 

The most crucial changes are in utils.py where i added two new functions to create two types of "group_by_queries" The output of these are then passed to the find_key_over_time() function through the new argument "group_queries".

### CODE REVIEW

File: vogue/server/utils.py [hunk 1]
```diff
@@ -59,48 +59,66 @@ def get_percentiles(samples: list, key: str)-> float:
 
     return percentiles
 
+def build_app_tag_group_queries()-> dict:
```
Comment (moonso) [inline vogue/server/utils.py]:
```suggestion
def build_app_tag_group_queries()-> list:
```
It says below that it returns a list of application tags, not a dict

File: vogue/server/utils.py [hunk 2]
```diff
@@ -59,48 +59,66 @@ def get_percentiles(samples: list, key: str)-> float:
...
     Input:
         title :         Plot title.
         year :          Data from this year will be shown in the plot. 
-        group_key :     Is the key in the database holding the group value.
+        group_queries : List of tuples (<group name>, <group query>)
         y_axis_key :    Key in database wich value will be plotted on the Y-axes (if given).
         y_unit :        Determines what to plot on the y axis (average/nr samples) (if "nr samples", 
                         no y_axis_key is needed).
         y_axis_label :  What it seems to be :)
         """
```
Comment (moonso) [inline vogue/server/utils.py]: Could you add `Returns` here and explain (with small example) what is returned?

Comment (ingkebil) [reply vogue/server/utils.py]: we should start using google-style docstring again: http://google.github.io/styleguide/pyguide.html#381-docstrings

**Comment by mayabrandi** [comment]
Will work on your suggestions straight ahead @moonso

**Pull request merged**